src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -113,21 +99,34 @@ interface PersonPhoneNumbers {
   toDelete: PersonPhoneNumberInput[];
 }
 
+export interface ModalState {
```

Comment:
@canac could you help me review this PR? I'm getting a little lost with how this is both using Formik and useState.